core/src/OpenMPTarget/Kokkos_OpenMPTarget_Abort.hpp

```diff
@@ -53,6 +53,7 @@ namespace Impl {
 
 KOKKOS_INLINE_FUNCTION void OpenMPTarget_abort(char const *msg) {
   fprintf(stderr, "%s.\n", msg);
+  fflush(stderr);
```

Comment:
Is this necessary? I thought `stderr` is unbuffered.

Reply:
Will revert.  I copied from `host_abort`